app/views/layouts/_header.html.erb

```diff
@@ -70,6 +70,17 @@
 </div>
 
 <script type="text/javascript">
+  document.addEventListener("DOMContentLoaded", function () {
+    var gettingStartedLink = document.querySelector("#getting-started-dropdown");
+    gettingStartedLink.addEventListener("click", function () {
+      gettingStartedLink.classList.toggle("rotate180");
+    });
+    document.addEventListener("click", function () {
+     if (gettingStartedLink.classList.contains("rotate180")) {
+       gettingStartedLink.classList.remove("rotate180");
+     }
```

Comment:
```suggestion
     gettingStartedLink.classList.remove("rotate180");
```
You don't need to check if it contains, as .remove method doesn't raise error if class not exists